Libraries/Animated/src/NativeAnimatedHelper.js

```diff
@@ -243,13 +219,11 @@ function validateTransform(
 }
 
 function validateStyles(styles: {[key: string]: ?number, ...}): void {
```

Comment:
I think we can delete this completely, I highly doubt there are other external callsites, kind of a private api anyway.

Reply:
I already added a deprecation warning so that we can remove this one safely at a later time - would that be ok?

Reply:
I think we can delete it now since I don't see why it would be called from user code. Is there a case for it I don't see?

Reply:
I'm not sure - but it is possible to call this from user land - so we don't know if anyone is using this or not. That's why I suggest we go the deprecation route by giving a warning and then remove in a later release.

Reply:
Do you want me to remove or can we keep with deprecation warning, @janicduplessis? If we can keep it I think the branch is ready to be merged.

Reply:
Let's remove it

Reply:
Done!